Trimble.Modus.Components/Controls/TMCard/TMCard.xaml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<ContentView xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="Trimble.Modus.Components.TMCard"
+             x:Name="card">
+  <Border x:Name="border"
+          BindingContext="{x:Reference card}"
+          Padding="{Binding Padding}">
+    <Border.Shadow>
+      <Shadow Brush="{DynamicResource CardShadowColor}"
+              Radius="{OnPlatform Default=3,Android=15}"
+              Opacity="0.6" />
+    </Border.Shadow>
+    <Border.StrokeShape>
```

Comment:
Toggling the 'Selected' property does not change the appearance. It still says in blue color. Can you verify once?

Reply:
Fixed

Reply:
Maybe you should try VisualStateManager to manage the 'Selected' state.

Reply:
This seems to be fixed now